lib/magma.rb

```diff
@@ -42,6 +40,13 @@ def setup_db
     @db.extension :connection_validator
     @db.extension :pg_json
     @db.pool.connection_validation_timeout = -1
+
+    Sequel::Model.plugin :timestamps, update_on_create: true
+    Sequel::Model.require_valid_table = false
+    Sequel.extension :inflector
+
```

Comment:
Will this play well with ActiveSupport, which also tries to do pluralization? I've run into trouble here before due to confusion over who, exactly, is determining what is "plural" at any given time, and differing ideas of the correct pluralization.

Reply:
Hmm...lemme play around with this a little bit. I didn't run into any problems when I was building this out and testing the changes.